### PR TITLE
gsg: add how-to guide

### PR DESCRIPTION
**Pull request by ttzeng**
This commit incorporates Pull Requests #54, #55, #56 as the central page of the development tips for the Celadon project.

Signed-off-by: Ming Tan <ming.tan@intel.com>
Signed-off-by: Tonny Tzeng <tonny.tzeng@intel.com>
Reviewed-by: Kristi Rifenbark <kristi@buzzcollectivemarketing.com>

### CODE REVIEW

**Comment by ttzeng** [comment]
@tanminger, I'd like to replace your 3 PRs with this commit, please take a look and review.

**Comment by ttzeng** [comment]
@klynnrif, I've incorporated 3 PRs to this one. Your preview review changes on #54 have been applied to the 1st section of this article, please help to review the content continue on the 2nd section onward. Thanks in advance. 👍

File: source/getting_started/how-to.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    .. note::
+        #. If you already installed Android in the internal storage, disable the internal storage if you are using USB live boot.
+        #. If you want to use one USB disk in two computers, make sure the lock status is same in each computer.
+
+How to dual boot |C| and Windows
+--------------------------------
+
+You may want to install Windows and |C| Anroid on one device, and select which OS to boot everytime the device is powering up. To achieve the dual boot, you need to install Windows and |C| on two storage devices seperately. There are two approaches:
+
+#. Use Kernel Flinger UEFI executables:
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Is "Kernel Flinger" supposed to be two words? If no, I suggest combining here and the instance in line 71.

File: source/getting_started/how-to.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    The Kernel Flinger UEFI installer does not prompt for the target storage device to install. You need to disconnect the Windows installed storage device first, connect the storage back to the device after installing |C|, and then select which OS to boot from the BIOS boot menu.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: The Kernelflinger UEFI installer does not prompt the target storage device to install. Disconnect the Windows installed storage device first, connect the storage back to the device after installing |C|, and then select which OS to boot from the BIOS boot menu.

File: source/getting_started/how-to.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+#. Install using GRUB installer image:
+
+    A GRUB installer image is created if you built the image with the following command. The GRUB installer provides interactive prompt for you to select which storage device to install |C|.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Suggested rewrite: 
A GRUB installer image is created if you build the image with the following command. The GRUB installer provides an interactive prompt for you to select which storage device to install |C|.

File: source/getting_started/how-to.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+    .. code-block:: bash
+
+        $ make SPARSE_IMG=true project_celadon-efi -j $(nproc)
+
+    Substitute *$(nproc)* to the appropriate value according to the CPU cores on your build server.
+
+    After installing |C| Android and Windows, you can select which OS to boot from the BIOS boot menu. Or, you can add a boot item to the GRUB menu for Windows, and always boot into the GRUB bootloader and select which OS to boot from the GRUB menu.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Suggested rewrite: 
After installing |C| Android and Windows, select which OS to boot from in the BIOS boot menu. Alternatively, add a boot item to the GRUB menu for Windows, and always boot into the GRUB bootloader. Then you can select which OS to boot from in the GRUB menu.

File: source/getting_started/how-to.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+How to change the partition size in |C| image
+---------------------------------------------
+
+An Android system typically comprises multiple disk partitions, each parition is associated with an *\*.img* file in the |C| image. In addition, |C| also supports a whole GPT image to be flashed to the entire disk.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Suggested rewrite: 
An Android system is typically comprised of multiple disk partitions. Each parition is associated with an *\*.img* file in the |C| image. In addition, |C| supports a whole GPT image to be flashed to the entire disk.

File: source/getting_started/how-to.rst [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+Some partitions, such as the system partition, are huge, but some of them are empty, the **SPARSE** image format is used to reduce the image size.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Suggested rewrite: 
Some partitions (e.g. the system partition) are huge, but some of them are empty. The **SPARSE** image format is used to reduce the image size.

File: source/getting_started/how-to.rst [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+    .. list-table::
+
+        * - device/intel/mixins/groups/boot-arch/project-celadon/gpt.ini
+        * - device/intel/mixins/groups/boot-arch/project-celadon/option.spec
+        * - device/intel/mixins/groups/vendor-partition/true/option.spec
+        * - device/intel/mixins/groups/config-partition/true/gpt.ini
+        * - device/intel/mixins/groups/factory-partition/true/gpt.ini
+
+You can modify *device/intel/project-celadon/[lunch_target]/mixins.spec* to change some of the default settings.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Suggested rewrite: 
Modify *device/intel/project-celadon/[lunch_target]/mixins.spec* to change some of the default settings.

File: source/getting_started/how-to.rst [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+.. note::
+    1. Changing the partition size, will cause the OTA update failure.
```
Comment (klynnrif) [inline source/getting_started/how-to.rst]: Remove comma: Changing the partition size will cause the OTA update failure.

File: source/getting_started/how-to.rst (hunk 1 above)
Comment (ttzeng) [reply source/getting_started/how-to.rst]: > Is "Kernel Flinger" supposed to be two words? If no, I suggest combining here and the instance in line 71.

Thanks @klynnrif, good catch! checked our [previous guide](https://01.org/projectceladon/documentation/getting_started/install-run), it should be two words for consistency.

File: source/getting_started/how-to.rst (hunk 8 above)
Comment (ttzeng) [reply source/getting_started/how-to.rst]: @klynnrif, thank you for the great support, always! 🥇

**Pull request merged**